src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -674,6 +678,36 @@ class FrontendGenImpl {
       mlir::emitWarning(UnknownLoc(),
           "Could not find op importer: assuming this "
           "represents a custom operator.");
+    mlir::emitWarning(UnknownLoc(), "Could not find op importer: assuming this "
+                                    "represents a custom operator.");
+    int nOps = node.input().size();
+    auto funcName = opName.str();
+    std::vector<mlir::Type> outputTypes;
+    std::vector<mlir::Value> inputs;
+    auto attributes = ImportNodeAttributes(node);
+    auto mlirAttr = builder_.getStringAttr(funcName);
+    auto funcAttr = builder_.getNamedAttr("function_name", mlirAttr);
+    attributes.push_back(funcAttr);
+    //auto customOp = builder_.create<ONNXCustomOp>(UnknownLoc(), outputTypes, inputs, attributes);
+    int nIn = 0;
+    int nOut=0;
+    //mlir::Value customResult = *(customOp.getODSResults(0).begin());
+    //std::vector<mlir::Value> inputs;
+    for (const auto &item : node.input()) {
+      ++nIn;
+      if (initializedTensors.ContainKey(legalize_name(item))) {
```

Comment:
Nit: it may be worthwhile extracting the logic in lines 424 to 432 as a reusable method and call it here. That will also help handle optional inputs. It looks like this logic is repeated in several places.

Reply:
done